cli/src/commands/git/push.rs

```diff
@@ -786,16 +795,12 @@ fn classify_bookmark_update(
                 "Run `jj bookmark track {remote_symbol}` to import the remote bookmark."
             )),
         }),
-        // TODO: deprecate --allow-new and make classify_bookmark_push_action()
-        // reject untracked remote?
```

Comment:
nit: this comment is still valid. Internal implementation relies on `allow_new` flag to push newly-created `--change` bookmarks for example.